### PR TITLE
API reference migration

### PR DESCRIPTION
**Pull request by ShreyaThapa**

### CODE REVIEW

File: pages/api-reference/authorization/index.mdx [hunk 1]
```diff
@@ -9,8 +9,8 @@ guide:
   step: 0
   icon: authorization.svg
 meta:
-  title: API Reference | Dwolla API Documentation
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: API Reference | Dwolla Authoriztion Overview
```
Comment (ShreyaThapa) [inline pages/api-reference/authorization/index.mdx]: Typo in "Authorization"

File: pages/api-reference/customers/index.mdx [hunk 2]
```diff
@@ -9,8 +9,8 @@ guide:
   step: 0
   icon: authorization.svg
 meta:
-  title: API Reference | Dwolla API Documentation
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: API Reference | Customer Overview
+  description: "Endpoints for Customers that include Customer creation and search, business classifications, and customer updating."
```
Comment (ShreyaThapa) [inline pages/api-reference/customers/index.mdx]: Perhaps we could change this to `"Create, search and update Customers, and search for business-classification."`

File: pages/api-reference/accounts/create-a-funding-source-for-an-account.mdx [hunk 3]
```diff
@@ -4,8 +4,7 @@ title: "Create a funding source for an account"
 guide:
   step: 2
 meta:
-  title: "API Reference | Dwolla API Documentation"
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: "API Reference | Create a Funding Source for an Account"
```
Comment (ShreyaThapa) [inline pages/api-reference/accounts/create-a-funding-source-for-an-account.mdx]: Should we use the title schema `"API Reference | * "` on `index.mdx` pages and `" * | Dwolla API Documentation"` on the rest?

File: pages/api-reference/funding-sources/index.mdx [hunk 4]
```diff
@@ -9,8 +9,8 @@ guide:
   step: 0
   icon: authorization.svg
 meta:
-  title: API Reference | Dwolla API Documentation
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: API Reference | Funding Sources
+  description: "Create, update, remove, or retrieve, a funding source with the API. Use Intsant Acctount Verification (IAV) or micro depositis to verify the funding source."
```
Comment (ShreyaThapa) [inline pages/api-reference/funding-sources/index.mdx]: We can remove a comma in this line. 
Change to - `"Create, update, remove, or retrieve a funding source with the API."`

File: pages/api-reference/labels/create-a-label-ledger-entry.mdx [hunk 5]
```diff
@@ -4,8 +4,7 @@ title: "Create a label ledger entry"
 guide:
   step: 3
 meta:
-  title: "API Reference | Dwolla API Documentation"
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: "Create a Label Ledger | Dwolla API Documentation"
```
Comment (ShreyaThapa) [inline pages/api-reference/labels/create-a-label-ledger-entry.mdx]: Let's rephrase this to `"Create a Label Ledger Entry"`

File: pages/api-reference/accounts/list-and-search-transfers-for-an-account.mdx [hunk 6]
```diff
@@ -4,8 +4,7 @@ title: "List and search transfers for an account"
 guide:
   step: 4
 meta:
-  title: "API Reference | Dwolla API Documentation"
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: "API Reference | List and Search Transfers for an Account"
```
Comment (ShreyaThapa) [inline pages/api-reference/accounts/list-and-search-transfers-for-an-account.mdx]: Change to `"List and Search Transfers for an Account | Dwolla API Documentation"`

File: pages/api-reference/accounts/list-funding-sources-for-an-account.mdx [hunk 7]
```diff
@@ -4,8 +4,7 @@ title: "List funding sources for an account"
 guide:
   step: 3
 meta:
-  title: "API Reference | Dwolla API Documentation"
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: "API Reference | List Funding Sources for an Account"
```
Comment (ShreyaThapa) [inline pages/api-reference/accounts/list-funding-sources-for-an-account.mdx]: Change to `"List Funding Sources for an Account | Dwolla API Documentation"`.

File: pages/api-reference/accounts/list-masspayments-for-an-account.mdx [hunk 8]
```diff
@@ -4,8 +4,7 @@ title: "List mass payments for an account"
 guide:
   step: 5
 meta:
-  title: "API Reference | Dwolla API Documentation"
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: "API Reference | List Mass Payments for an Account"
```
Comment (ShreyaThapa) [inline pages/api-reference/accounts/list-masspayments-for-an-account.mdx]: Change to `"List Mass Payments for an Account | Dwolla API Documentation"`

File: pages/api-reference/accounts/retrieve-account.mdx [hunk 9]
```diff
@@ -4,8 +4,7 @@ title: "Retrieve Account Details"
 guide:
   step: 1
 meta:
-  title: "API Reference | Dwolla API Documentation"
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: "API Reference | Retrieve an Account"
```
Comment (ShreyaThapa) [inline pages/api-reference/accounts/retrieve-account.mdx]: Change to `"Retrieve an Account | Dwolla API Documentation"`

File: pages/api-reference/drop-in-components/index.mdx [hunk 10]
```diff
@@ -9,8 +9,8 @@ guide:
   step: 0
   icon: authorization.svg
 meta:
-  title: API Reference | Dwolla API Documentation
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: API Reference | Dwolla API Documentation — Drop-in Components
```
Comment (ShreyaThapa) [inline pages/api-reference/drop-in-components/index.mdx]: Change to `"API Reference | Drop-in Components"`

File: pages/api-reference/events/index.mdx [hunk 11]
```diff
@@ -9,8 +9,8 @@ guide:
   step: 0
   icon: authorization.svg
 meta:
-  title: API Reference | Dwolla API Documentation
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: API Reference | Dwolla API Documentation — Events
```
Comment (ShreyaThapa) [inline pages/api-reference/events/index.mdx]: Change to `"API Reference | Events"`

File: pages/api-reference/webhook-subscriptions/index.mdx [hunk 12]
```diff
@@ -9,8 +9,8 @@ guide:
   step: 0
   icon: authorization.svg
 meta:
-  title: API Reference | Dwolla API Documentation
-  description: "Complete reference documentation for the Dwolla API. Includes resource descriptions, endpoints and methods, parameters, and request and response examples."
+  title: API Reference | Dwolla API Documentation — Webhook Subscriptions
```
Comment (ShreyaThapa) [inline pages/api-reference/webhook-subscriptions/index.mdx]: Change to `"API Reference | Webhook Subscriptions"`

File: pages/api-reference/authorization/index.mdx [hunk 13]
```diff
@@ -0,0 +1,32 @@
+---
+layout: guides
+title: "Authorization"
```
Comment (spencerhunter) [inline pages/api-reference/authorization/index.mdx]: Should we change this to "Overview", similar to the other sections?

File: pages/api-reference/accounts/retrieve.mdx [hunk 14]
```diff
@@ -0,0 +1,98 @@
+---
+layout: guides
+title: "Retrieve Account Details"
```
Comment (spencerhunter) [inline pages/api-reference/accounts/retrieve.mdx]: can we convert this to sentence case? similar to the other subsections

File: pages/api-reference/webhooks/list-retries.mdx [hunk 15]
```diff
@@ -0,0 +1,97 @@
+---
+layout: guides
+title: "List retries for a webhookn"
```
Comment (spencerhunter) [inline pages/api-reference/webhooks/list-retries.mdx]: small typo. Remove the "n" at the end of webhook
